src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp

```diff
@@ -51,42 +51,61 @@ void CobConverter::UpdateState(uint64_t CurrentSimNanos)
 {
     CameraConfigMsgPayload cameraSpecs = this->cameraConfigInMsg();
     OpNavCOBMsgPayload cobMsgBuffer = this->opnavCOBInMsg();
-    OpNavUnitVecMsgPayload uVecMsgBuffer = this->opnavUnitVecOutMsg.zeroMsgPayload;
```

Comment:
You must know what I'm going to say about this commit ;)

Reply:
Split the commit up into several commits.